actors/miner/src/types.rs

```diff
@@ -490,8 +490,8 @@ pub struct ProveReplicaUpdates3Params {
     pub aggregate_proof: RawBytes,
     // The proof type for all sector update proofs, individually or before aggregation.
     pub update_proofs_type: RegisteredUpdateProof,
-    // The proof type for the aggregate proof (ignored if no aggregate proof).
-    pub aggregate_proof_type: RegisteredAggregateProof,
+    // The proof type for the aggregate proof (must be None if no aggregate proof).
```

Comment:
Nit: This isn't actually true today, afaic tell -- this field is simply ignored in all cases (cuz we don't actually support aggregated PRUs yet), and so you can put whatever you want in here?

Reply:
Right, I'll fix so it is enforced, so that doesn't break anyone in the future when we need to.